### DISCUSSION TITLE
How do I publish my GitHub page to Google?

### DISCUSSION THREAD

**Discussion by MiniSkoot32** [community/community#114386] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I started a website for my company hosted on GitHub, but I can't figure out how to verify it. My link is [pywaresolutions.github.io](url) and have tried using the html page, but I got an error. I tried the same for a html tag, and it didn't work. If you know how to fix it, I would appreciate it if you could. Thanks in advance!

**Comment by r4wand** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> I started a website for my company hosted on GitHub, but I can't figure out how to verify it. My link is [pywaresolutions.github.io](url) and have tried using the html page, but I got an error. I tried the same for a html tag, and it didn't work. If you know how to fix it, I would appreciate it if you could. Thanks in advance!

What do you mean by publishing to Google? Do you mean you can't find the website when doing a Google search?

Your website is live at  https://pywaresolutions.github.io but it seems to be broken.

**Reply by madhatter** [reply]
Meanwhile it seems to be available again, but the link in the post is broken.

**Reply by MiniSkoot32** [reply]
I am trying to get it on GOogle via the Google Search Console, because I read something that google does not index GitHub pages. @madhatter how is my site broken?

**Reply by madhatter** [reply]
When I click the link in your initial post it directs me to https://github.com/orgs/community/discussions/url
where I see the 404 page of Github.

**Reply by MiniSkoot32** [reply]
mine takes me to the website I made

**Reply by MiniSkoot32** [reply]
so how can I successfully my GitHub page onto google using the Google search console, @madhatter @r4wand

**Reply by devcavin** [reply]
Just type a word related to your web page "pywaresolutions.github.io" ie 'pywaresolutions' the browser is going to point out to links to your website. Check the image below I just searched and found a direct link to it as the first output...

![Screenshot 2024-04-03 013607](https://github.com/community/community/assets/114779273/7dca751b-4777-4c95-95b3-e3bc93a2d672)

**Comment by jfullstackdev** [comment]


**Reply by ebndev** [reply]
Hi @jdevfullstack, 

We’ve clarified our stance on using generative AI tools within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by MiniSkoot32** [comment]
Thanks for helping
